dev.ftb.ftb-app.yml

```diff
@@ -0,0 +1,51 @@
+app-id: dev.ftb.ftb-app
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: ftb.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --env=CHROME_WRAPPER=zypak-wrapper
+  - --filesystem=xdg-download:create # Import/export modpacks and logs
+  - --filesystem=xdg-desktop:create # Import/export modpacks and logs
+  - --filesystem=~/.ftba:create # Home folder for the FTB App
```

Comment:
why not use persist here?

Reply:
We have a couple of issues with our backend for the application that means it's not able to figure out how to open this path, we're not really sure why and we're not really sure how to know if we're in a flatpak or not :P 

This would also break compatibility if anyone migrated from an existing linux build (AppImage, deb, etc) as this is the exact path all of their instances will be (Mods, configs, worlds, etc)